### PR TITLE
update to core v6 beta 3

### LINKED ISSUE #22: Update NUnit to version 3
See https://github.com/Particular/PlatformDevelopment/issues/792

This is small enough to fall under Maintainer Prio

### PR DESCRIPTION
**Pull request by SimonCropp**
-update nunit

fixes #22

### CODE REVIEW

**Comment by SimonCropp** [comment]
blocked by https://github.com/Particular/NServiceBus/pull/3775

**Comment by danielmarbach** [comment]
Merged PR in core

**Comment by SimonCropp** [comment]
@Particular/container-maintainers this now looks good. merge please

File: src/NServiceBus.Spring/SpringObjectBuilder.cs [hunk 1]
```diff
@@ -157,11 +157,11 @@ void ThrowIfAlreadyInitialized()
         {
             if (initialized)
             {
-                throw new InvalidOperationException("You can't alter the registrations after the container components has been resolved from the container");
+                throw new InvalidOperationException("Alter the registrations after the container components has been resolved from the container is not possible.");
```
Comment (boblangley) [inline src/NServiceBus.Spring/SpringObjectBuilder.cs]: I think this should this read `Altering the registrations...` instead of `Alter the registrations`.

**Comment by boblangley** [comment]
Merged #23

**Pull request merged**